Package.swift

```diff
@@ -35,7 +35,10 @@ let package = Package(
         .macOS(.v10_15), .iOS(.v12), .tvOS(.v12)
     ],
     products: [
-        .library(name: "AzureCore", targets: ["AzureCore"])
+        .library(name: "AzureCore", targets: ["AzureCore"]),
```

Comment:
Can we remove the library here that ends with "Dynamic" and keep the rest as it is

Reply:
We have opened PR # 447 to address this comment.